joomla/joomla-cms
# [4.0] Remove jQuery Migrate

### Summary of Changes

This PR removes jQuery migrate from Joomla

### Testing Instructions

- Apply patch
- Ensure `jquery.migrate.min.js` doesn't get loaded.

With a hard refresh on Chrome, this alone shaves off **30ms** off the loading time
